src/contracts/yas_pool.cairo

```diff
@@ -429,10 +461,91 @@ mod YASPool {
             self.unlock();
             (amount_0, amount_1)
         }
+
+        fn mint(
+            ref self: ContractState,
+            recipient: ContractAddress,
+            tick_lower: i32,
+            tick_upper: i32,
+            amount: u128,
+            data: Array<felt252>
+        ) -> (u256, u256) {
+            self.check_and_lock();
+
+            assert(amount > 0, 'amount must be greater than 0');
+            let (_, amount_0, amount_1) = self
```

Comment:
If we are not using the first element returned, why are returning in it? This function is only used in `mint` right?

Reply:
The `modify_position` function is used in the `burn` function and uses the first field